js/transform.js

```diff
@@ -222,6 +232,15 @@ function displayForm(e) {
 }
 
 function generateAndDisplayForm(formId, dest) {
+    const INTEGER_AFTER_UNDERSCORE_REGEX = /(.*)_([0-9]+)/;
+    const match = formId.match(INTEGER_AFTER_UNDERSCORE_REGEX);
+    if (match) {
+        var formName = match[1]
+        var formIterator = match[2]
```

Comment:
[Pebble] Change `var` to `const` if they will not be reassigned. Separately, now I'm confused why we look at the second and third elements vs. first and second! Could you explain that part?

Reply:
`match[0]` returns the full string that was matched (e.g. IRS996_0), `match[1]` and `match[2]` return the first (IRS996) and second (0) capture group respectively! 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match
> If the g flag is not used, only the first complete match and its related capturing groups are returned. In this case, match() will return the same result as [RegExp.prototype.exec()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/exec) (an array with some extra properties).

RegExp Docs: http://devdoc.net/web/developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/exec.html